### PR TITLE
initial gw-sklearn-xarray

### PR DESCRIPTION
**Pull request by mmann1123**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mmann1123** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes, ready for review

**Comment by jgrss** [comment]
I am OK with being a maintainer of this [sklearn-xarray fork](https://github.com/jgrss/sklearn-xarray).

File: recipes/gw-sklearn-xarray/meta.yaml
Comment (dopplershift) [inline recipes/gw-sklearn-xarray/meta.yaml:1]: Should this be "gw-sklearn-xarray"? Otherwise, this conflicts with the name on PyPI from the base of the fork.

File: recipes/gw-sklearn-xarray/meta.yaml
Comment (dopplershift) [inline recipes/gw-sklearn-xarray/meta.yaml:13]: It looks like this is pure Python and could be noarch:
```suggestion
  noarch: python
```

File: recipes/gw-sklearn-xarray/meta.yaml
Comment (dopplershift) [inline recipes/gw-sklearn-xarray/meta.yaml:26]: For noarch:
```suggestion
  host:
    - python >=3.7
    - pip
```

File: recipes/gw-sklearn-xarray/meta.yaml
Comment (dopplershift) [inline recipes/gw-sklearn-xarray/meta.yaml:28]: For noarch:
```suggestion
    - python >=3.7
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

File: recipes/gw-sklearn-xarray/meta.yaml
Comment (dopplershift) [inline recipes/gw-sklearn-xarray/meta.yaml:1]: Just a last clarification before we merge. The title of the PR says "gw-sklearn-xarray", while here we have "sgw-sklearn-xarray". Can you confirm name is set correctly here or otherwise make any needed correction?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
